File: src/main/java/org/pixeltime/enchantmentsenhance/interfaces/GlowItem.java
package org.pixeltime.enchantmentsenhance.interfaces;

import org.bukkit.inventory.ItemStack;

public interface GlowItem {
    /**
     * Makes an item glowing.
     *
     * @param item
     * @return
     */
     ItemStack addGlow(ItemStack item);
}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/interfaces/GlowItem.kt
package org.pixeltime.enchantmentsenhance.interfaces

import org.bukkit.inventory.ItemStack

interface GlowItem {
    /**
     * Makes an item glowing.
     *
     * @param item
     * @return
     */
    fun addGlow(item: ItemStack): ItemStack
}
